WTWSMS/common/nicknames/00_nicknames.txt

```diff
@@ -1881,6 +1902,9 @@ nick_the_black = {
 			trait = excommunicated
 			trait = impaler
 			trait = bad_priest_christian
+			trait = cannibal_trait
+			has_character_modifier = cannibal
+			has_character_modifier = crazy_cannibal
```

Comment:
You don't think a cannibal has a black heart? Cannibalism, especially of innocents, is a crime in the eyes of man and God. Hence, being known for their black heart.